website/docs/guides/migration/versions/04-upgrading-to-v1.4.md

```diff
@@ -27,7 +27,16 @@ Relevant changes:
 
 ### For maintainers of adapter plugins
 
-Link to GH discussion forthcoming
+> **TL;DR** Not much heavy lifting for this minor version. But FYI there will be more work for `1.5.0`. Be warned!
+
+The high-level changes are:
+- Add support for Python 3.11
+- Rename/Replace deprecated exception functions
+- Consolidate timestamp functions & macros
```

Comment:
I think this change was actually included in v1.3. It landed in the v1.4 changelog because it was merged during v1.3 RC period